src/components/VisibleRender/VisibleRender.md

```diff
@@ -0,0 +1,30 @@
+### Examples
+
+Basic usage:
+
+```js { "props": { "data-example": "basic" } }
+const { GutterSize, TextSize } = require('../Block');
+
+const renderInView = () => {
+  return (
+    <div style={{ backgroundColor: '#adedba', transition: 'background-color 1s' }}>
+      <Block padding={GutterSize.XXLARGE} textSize={TextSize.LARGE}>In view</Block>
+    </div>
+  );
+};
+
+const renderOutOfView = (inView) => {
+  return (
+    <div style={{ backgroundColor: '#f98985', transition: 'background-color 1s' }}>
+      <Block padding={GutterSize.XXLARGE} textSize={TextSize.LARGE}>Out of view</Block>
+    </div>
+  );
+};
+
+<VisibleRender 
+  whenInView={renderInView}
+  whenOutOfView={renderOutOfView}
+  onEnter={action('scrolled into view')}
+  onLeave={action('scrolled out of view')}
```

Comment:
Should these have `View` in the name to be clearer, and to create a connection with the other two props? `onEnterView` and `onLeaveView`? Or `onInView` and `onOutOfView`, but they sound worse...

Reply:
will rename